templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -273,8 +273,14 @@ contents:
     }
 
     verify_certs() {
+      iterations=0
       while [ "$(ls $ETCD_STATIC_RESOURCES | wc -l)" -lt 9  ]; do
-        echo "Waiting for certs to generate.."
+        let iterations=$iterations+1
+        if [ $iterations -gt 60 ]; then
```

Comment:
question:
- How much time does it take for 60 iterations?

- Should the user know about the timeout so they don't kill the script manually? Maybe something like "Waiting N minutes for certs to generate.."

Reply:
Actually I guess you addressed this

> Waiting for certs to generate... ($iterations/60)

Reply:
Yeah. Thanks. If there is a different way you want me to tackle it, I can convert it into seconds and minutes too.

Reply:
10 mins which I believe is the timeout for the client